log4j-core/src/main/java/org/apache/logging/log4j/core/async/RingBufferLogEventTranslator.java

```diff
@@ -124,6 +129,11 @@ public void setBasicValues(
         this.nanoClock = aNanoClock;
     }
 
+    /**
+     * @deprecated since 2.25.0. {@link RingBufferLogEventTranslator} instances should only be used on the thread that
+     * created it.
+     */
+    @Deprecated
```

Comment:
Since we removed `AsyncLogger::initTranslatorThreadValues`, usages of this method will result in undefined behavior. Shouldn't we throw an `UnsupportedOE` in this method?

Reply:
I don't believe this method is used outside our codebase, but there are a couple of potential use cases: for example, if `RingBufferLogEventTranslator` instances are stored in a queue instead of being thread-local, calling `updateThreadValues` might be necessary to prepare the translator for use.

I would rather keep the method's implementation as it is now.